jivelite-sp

```diff
@@ -4,22 +4,57 @@ export LOG=/var/log/pcp_jivelite.log
 
 if [ -f /usr/local/etc/pcp/pcp.cfg ]; then
     source /usr/local/etc/pcp/pcp.cfg
-    if [ ! -z ${SCREENROTATE} ]; then
-        MODEL=$(busybox awk '/^Model/ { print $5 }' /proc/cpuinfo)
-        if [ "${MODEL}" == "5" ]; then
-            # Rotation CCW=270, UD=180, CW=90 degrees
-            if [ "${SCREENROTATE}" == "180" ]; then
-                export SDL_VIDEO_FBCON_ROTATION=UD
-                export TSLIB_CALIBFILE=/usr/local/etc/pointercal-r2
-                echo "Pi5 detected sdl screen rotation enabled." >> $LOG
-            fi
-        fi
-    fi	
 fi
 
+# Autodetect Touchscreen, and then mouse if no touch is found.
+TMPFILE=$(mktemp)
+udevadm info --export-db | grep "P: " | grep "event" | sed 's/P: //' > $TMPFILE
+for LINE in $(cat $TMPFILE); do
+   udevadm info --query=property --path=$LINE | grep -q TOUCH
+   [ $? -eq 0 ] && eventno=$(basename $LINE);TOUCH=1;MOUSE=0
```

Comment:
Could we do something like this?
``` 
  [ "$eventno" != "" ] &&  echo "Found Touch screen: $eventno" >> $LOG && break
```

Reply:
I added a bit more information rather than just the even number